packaging/installer/kickstart-ng.sh

```diff
@@ -0,0 +1,843 @@
+NETDATA_USER_CONFIG_DIR="/etc/netdata"
+NETDATA_AUTO_UPDATES="1"
+NETDATA_ONLY_NATIVE=0
+NETDATA_ONLY_STATIC=0
+
+NETDATA_DISABLE_TELEMETRY="${DO_NOT_TRACK:-0}"
+NETDATA_TARBALL_BASEURL="${NETDATA_TARBALL_BASEURL:-https://storage.googleapis.com/netdata-nightlies}"
+
+if [ ! -t 1 ]; then
+  INTERACTIVE=0
+else
+  INTERACTIVE=1
+fi
+
+# ======================================================================
+# Usage info
+
+usage() {
+  cat << HEREDOC
+USAGE: kickstart.sh [options]
+       where options include:
+
+  --dont-wait                Do not prompt for user input. (default: prompt if there is a controlling terminal)
+  --non-interactive          Do not prompt for user input.
+  --dont-start-it            Do not start the agent by default (only for static installs or local builds)
+  --interactive              Prompt for user input even if there is no controlling terminal.
+  --stable-channel           Install a stable version instead of a nightly build (default: install a nightly build)
+  --nightly-channel          Install a nightly build instead of a stable version
+  --no-updates               Do not enable automatic updates (default: enable automatic updates)
+  --auto-update              Enable automatic updates.
+  --disable-telemetry        Opt-out of anonymous statistics.
+  --native-only              Only install if native binary packages are available.
+  --static-only              Only install if a static build is available.
+  --reinstall                Explicitly reinstall instead of updating any existing install.
+  --claim-token              Use a specified token for claiming to Netdata Cloud.
+  --claim-rooms              When claiming, add the node to the specified rooms.
+  --claim-only               If there is an existing install, only try to claim it, not update it.
+  --claim-*                  Specify other options for the claiming script.
+  --no-cleanup               Don't do any cleanup steps. This is intended to help with debugging the installer.
+
+Additionally, this script may use the following environment variables:
+
+  TMPDIR:                    Used to specify where to put temporary files. On most systems, the default we select
+                             automatically should be fine. The user running the script needs to both be able to
+                             write files to the temporary directory, and run files from that location.
+  ROOTCMD:                   Used to specify a command to use to run another command with root privileges if needed. By
+                             default we try to use sudo, doas, or pkexec (in that order of preference), but if
+                             you need special options for one of those to work, or have a different tool to do
+                             the same thing on your system, you can specify it here.
+  DO_NOT_TRACK               If set to a value other than 0, behave as if \`--disable-telemetry\` was specified.
```

Comment:
We do we need this considering that we have the `--disable-telemetry` option?

Reply:
Because it’s a de-facto standard environment variable for this purpose. If we want to drop anything, I’d actually prefer to drop the option, but we can’t be cause we need to retain compatibility with the existing scripts.

Reply:
Hi @vkalintiris 

This variable was an external proposal (https://consoledonottrack.com/), aiming to be respected by various projects. Target being that a user may set it globally on his machine, without explicitly disabling telemetry on various software. It is supported like that in other parts besides kickstart too.

Reply:
Based on that website, it's been adopted by just 1 project other than us. All the PRs mentioned in homebrew, gatsby, netlify, syncthing and dotnet have been rejected/closed for reasons similar to the ones I've described in my previous comment.

Reply:
Sometimes you need to take the first step ! :-) I kinda like the idea...

In any way, it could be dropped I guess from kickstart since probably will be picked up from other places, but I think we should leave it as an extra safeguard. As for in general use of `DO_NOT_TRACK` I guess it's beyond this PR's scope ?

Reply:
DO_NOT_TRACK discussion https://github.com/netdata/netdata/issues/7404#issuecomment-561552931

Reply:
LOL talk about things I would not like to come back from dead

Reply:
If we are going to support it, we should support it everywhere, and this _will_ matter since we will have telemetry directly from the new kickstart script.